packages/graphql/mixin.core.js

```diff
@@ -14,6 +15,25 @@ function exists(path) {
   }
 }
 
+function contextRelativeExternals(externals) {
+  return function(context, request, callback) {
+    if (externals.includes(request)) {
+      const absolutePath = require.resolve(request, {
+        paths: require.main.paths.concat(context),
+      });
+      if ((absolutePath.match(/node_modules/g) || []).length > 1) {
+        const relativePath = relative(
+          this.config.rootDir,
+          absolutePath
+        ).replace(/^node_modules\//, '');
```

Comment:
Rather than messing with a string here, it might be more robust to make the path relative against `path.join(rootDir, 'node_modules)`.